app/javascript/app/components/about/about-faq/about-faq-data.js

```diff
@@ -359,13 +359,16 @@ export const sectionsData = [
       },
       {
         type: 'html',
-        title: 'What methodology is used to collect and sort data on long-term strategies?  ',
+        title:
+          'What methodology is used to collect and sort data on long-term strategies?  ',
         answer:
-          '<p>The structured indicators used for presenting LTS information are adapted from several previous WRI papers, including <a href="https://www.wri.org/publication/early-insights-long-term-climate-strategies" rel="noopener noreferrer" target="_blank">Early Insights on Long-term Climate Strategies</a> and <a href="https://www.wri.org/publication/designing-G20-long-term-strategies" rel="noopener noreferrer" target="_blank">Long-term Low Greenhouse Gas Emission Development Strategies: Approaches and Methodologies for Their Design</a>.</p><p>All data collected in this database come strictly from the LTS submissions only. No evaluation of ambition has been undertaken. Rather the database is provided to facilitate transparency and understanding of countries' long-term strategies.</p> '
+          // eslint-disable-next-line quotes
+          `<p>The structured indicators used for presenting LTS information are adapted from several previous WRI papers, including <a href="https://www.wri.org/publication/early-insights-long-term-climate-strategies" rel="noopener noreferrer" target="_blank">Early Insights on Long-term Climate Strategies</a> and <a href="https://www.wri.org/publication/designing-G20-long-term-strategies" rel="noopener noreferrer" target="_blank">Long-term Low Greenhouse Gas Emission Development Strategies: Approaches and Methodologies for Their Design</a>.</p><p>All data collected in this database come strictly from the LTS submissions only. No evaluation of ambition has been undertaken. Rather the database is provided to facilitate transparency and understanding of countries' long-term strategies.</p>`
```

Comment:
As "countries'" has an apostrophe or single quote we need to change the whole text quotes so it's not processed as an end of the string. Also, some eslint disable comment was added to avoid internal linting warnings